File: Cosmos-Android/app/src/main/java/wannabit/io/cosmostaion/model/Node_Info.java
package wannabit.io.cosmostaion.model;

import com.google.gson.annotations.SerializedName;

public class Node_Info {
    @SerializedName("network")
    public String network;

    @SerializedName("version")
    public String version;

    @SerializedName("channels")
    public String channels;

    @SerializedName("moniker")
    public String moniker;


}

File: Cosmos-IOS/Cosmostation/Cosmostation/Model/NodeInfo.swift
//
//  NodeInfo.swift
//  Cosmostation
//
//  Created by 정용주 on 2021/03/20.
//  Copyright © 2021 wannabit. All rights reserved.
//

import Foundation

public struct NodeInfo {
    var network: String?
    var version: String?
    var channels: String?
    var moniker: String?
    
    init(_ dictionary: NSDictionary?) {
        self.network = dictionary?["network"] as? String
        self.version = dictionary?["version"] as? String
        self.channels = dictionary?["channels"] as? String
        self.moniker = dictionary?["moniker"] as? String
    }
    
}
